packages/dx-react-chart-demos/src/demo-sources/area-chart/material-ui/area.jsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  AreaSeries,
+  Title,
+  Legend,
+  Grid,
+} from '@devexpress/dx-react-chart-material-ui';
+import { withStyles } from '@material-ui/core/styles';
+import { Scale } from '@devexpress/dx-react-chart';
+import { scalePoint } from 'd3-scale';
+
+const data = [
+  { year: '2010', android: 67225, ios: 46598 },
+  { year: '2011', android: 179873, ios: 90560 },
+  { year: '2012', android: 310088, ios: 118848 },
+  { year: '2015', android: 539318, ios: 189924 },
+];
+
+const legendStyles = () => ({
+  root: {
+    display: 'flex',
+    margin: 'auto',
+    flexDirection: 'row',
+  },
+});
+const legendLabelStyles = () => ({
+  label: {
+    paddingTop: '8px',
```

Comment:
You can the `theme` object to add a padding. It will look like `paddingTop: theme.spacing.unit`.